README.md

```diff
@@ -1,5 +1,30 @@
 BreathingTasks_PsychoPy
 ==============
-PsychoPy code to display instructions for a 'Breath Hold' and a 'Cued Deep Breathing' task. 
---------------
+PsychoPy code to display visual instructions for a 'Breath Hold' and a 'Cued Deep Breathing' task. 
+
 Written with PsychoPy3.0.5
+
+Tips for using this code:
+--------------
+
+- It is designed to be used alongside recordings of end-tidal CO2 and O2 via a nasal cannula, during MRI, but can be used with other set-ups. 
+- Press '5' when it says 'waiting for scanner ...' to manually start the breathing task instructions. Or set this input to be whatever the MRI sends to indicate the first volume of data is being acquired so that your breathing task will be synchronized to the start of your scan. 
```

Comment:
It would be nice to add an example of this code, if you have it around - you could add it yourself and comment it in the real code, or add it here indicating where to add it.